src/components/transitions/UIFadeSlideTransition.vue

```diff
@@ -0,0 +1,131 @@
+<template>
```

Comment:
This works exceptionally well 👏👏

Reply:
I did it so we can have slide back and forth transition between pages. (the page component might have to be wrapped for this)
But this does not work with group transitions. I will get to it sometimes in the future.